bofire/models/diagnostics.py

```diff
@@ -0,0 +1,180 @@
+from typing import Optional, Sequence
+
+import numpy as np
+import pandas as pd
+from pydantic import root_validator, validator
+from sklearn.metrics import (
+    mean_absolute_error,
+    mean_absolute_percentage_error,
+    mean_squared_error,
+    r2_score,
+)
+
+from bofire.domain.util import PydanticBaseModel, is_numeric
+from bofire.utils.enum import RegressionMetricsEnum
+
+metrics = {
+    RegressionMetricsEnum.MAE: mean_absolute_error,
+    RegressionMetricsEnum.MSD: mean_squared_error,
+    RegressionMetricsEnum.R2: r2_score,
+    RegressionMetricsEnum.MAPE: mean_absolute_percentage_error,
+}
+
+
+class CVResult(PydanticBaseModel):
```

Comment:
Calling this `CvResult` increases readability at least for me. See what Google proposes  https://google.github.io/styleguide/jsguide.html#naming-camel-case-defined.